bindings/cloudflare/src/index.ts

```diff
@@ -152,7 +152,7 @@ export function recaptchaConfigFromEnv(env: Env): RecaptchaConfig {
     challengePageSiteKey: env.CHALLENGE_PAGE_SITE_KEY,
     enterpriseSiteKey: env.ENTERPRISE_SITE_KEY,
     recaptchaEndpoint: env.RECAPTCHA_ENDPOINT ?? DEFAULT_RECAPTCHA_ENDPOINT,
-    sessionJsInjectPath: env.SESSION_JS_INSTALL_PATH,
+    sessionJsInjectPath: env.SESSION_JS_INSTALL_PATH || env.RECAPTCHA_JS_INSTALL_PATH,
```

Comment:
Might not be relevant if my comment above is correct but if the path could be either session or not-session then the config attribute should be renamed as well.